pypeman/channels.py

```diff
@@ -455,6 +457,28 @@ def graph_dot(self, end=''):
         for end, sub in after:
             sub.graph_dot(end=end)
 
+    def jsonable_msg_info_for_admin(self, msg):
+        """ creates jsonable info representing the message for admin interfaces.
+            (Perhaps there's a better method name?)
+        """
+
+        encoder = JsonableEncoder()
+
+        result = {}
+        logger.debug("keys = %s", vars(self).keys())
```

Comment:
Can you make this message more clear or remove it ?

Reply:
suggest to remove